### PR TITLE
Updates/keytech

### PR DESCRIPTION
**Pull request by girpanns**
fix overlapping, udah beres mas

### CODE REVIEW

**Review by JordanAryaLeksana — APPROVED** [review]
lgtm

**Review by girpanns — COMMENTED** [review]
udah slsai mas

File: src/pages/KeyTech/index.jsx
Comment (JordanAryaLeksana) [inline src/pages/KeyTech/index.jsx:1]: lgtm

File: public/NLP_KeyTech.png
Comment (JordanAryaLeksana) [inline public/NLP_KeyTech.png:1]: lgtm

**Review by JordanAryaLeksana — APPROVED** [review]

**Review by JordanAryaLeksana — APPROVED** [review]

**Pull request closed without merging**